src/NUnitFramework/tests/Internal/EventListenerTextWriterTests.cs

```diff
@@ -79,7 +81,7 @@ public void TestWriteStringArg1()
             ListenerWriter.Write(format, arg0);
             ListenerWriter.WriteLine(format, arg0);
 
-            var expected = "20 Apr 2017";
+            var expected = $"20 {CultureInfo.CurrentCulture.DateTimeFormat.GetAbbreviatedMonthName(4)} 2017";
```

Comment:
If this is supposed to be CurrentCulture and not InvariantCulture, can we do `arg0.ToString("dd MMM yyyy")` or `$"{arg0:dd MMM yyyy}"` instead?